mlblocks/mlpipeline.py

```diff
@@ -767,6 +771,7 @@ def fit(self, X=None, y=None, output_=None, start_=None, debug=False, **kwargs):
             debug_info = defaultdict(dict)
             debug_info['debug'] = debug.lower() if isinstance(debug, str) else 'tmio'
 
+        early_stop = False
```

Comment:
`early_stop` sounds like an extraordinary thing but this functionality is just regular behavior.
I suggest inverting the meaning of the variable and calling it `fit_pending`: "If `True`, there are still blocks that are pending to be fitted"

Reply:
fixed